src/jdk.compiler/share/classes/com/sun/tools/javac/jvm/ClassFile.java

```diff
@@ -123,7 +123,8 @@ public enum Version {
         V62(62, 0),   // JDK 18
         V63(63, 0),   // JDK 19
         V64(64, 0),   // JDK 20
-        V65(65, 0);   // JDK 21
+        V65(65, 0),   // JDK 21
+        V66(66, 0);   // JDK 22
```

Comment:
A very small nit/suggestion - it should be possible to do:
```
V66(66, 0),   //JDK 22
;
```

(i.e. ending the enum constant with `,`, and putting the semicolon on a new line.) This way adding a new constant would mean just one line addition, no modification. (The same could be done for other enums.)

Reply:
Updated.